packages/TorneloScoresheet/src/types/GeneralSettingsState.ts

```diff
@@ -3,10 +3,17 @@ export enum ChessPieceStyles {
   CLASSIC,
 }
 
+export enum ChessBoardStyles {
```

Comment:
Are these being saved. They should have explict values if they are being saved in settings. 
Eg 
TORNELO = 'TORNELO',

Otherwise I believe they will be saved in the settings as 0 and 1.